xknx/remote_value/remote_value_climate_mode.py

```diff
@@ -121,3 +130,24 @@ def from_knx(self, payload):
             return None
         raise CouldNotParseTelegram("payload invalid",
                                     payload=payload, device_name=self.device_name, feature_name=self.feature_name)
+
+
+class RemoteValueBinaryHeatCool(_RemoteValueBinaryClimateMode):
```

Comment:
Do we need a test for this new class?

Reply:
Right! 
I miss the coveralls bot nagging at me for not writing tests 🤖

Reply:
Why does it not work anymore?

Reply:
I think netlify just checks PRs to the main branch. 
Coveralls is gone for some time. I don't know what happened there.